wolfhsm/wh_server.h

```diff
@@ -76,7 +76,7 @@ typedef struct {
         Cmac cmac[1];
 #endif
     };
-    union {
+    union crypto_context_pubkey {
```

Comment:
@jpbland1 can you think of better names for these? Doesn't matter that much since the name of the union will never be referenced, but it is a good opportunity for more self-documenting code